### PR TITLE
[ja] Translate tasks/debug-application-cluster/troubleshooting into Japanese

### LINKED ISSUE #30875: [ja] Translate tasks/debug-application-cluster/troubleshooting into Japanese
# This is a Feature Request

## What would you like to be added

Translate [tasks/debug-application-cluster/troubleshooting](https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/)  into Japanese.

## Why is this needed

This page has not been translated.

## Comments

/language ja
/assign

### PR DESCRIPTION
**Pull request by ptux**
resolve #30875

### CODE REVIEW

File: content/ja/docs/tasks/debug-application-cluster/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+
+Once you are registered, browse the growing list of channels for various subjects of
+interest. For example, people new to Kubernetes may also want to join the
+[`#kubernetes-novice`](https://kubernetes.slack.com/messages/kubernetes-novice) channel. As another example, developers should join the
+[`#kubernetes-dev`](https://kubernetes.slack.com/messages/kubernetes-dev) channel.
+
+登録が完了したら、増え続けるチャンネルリストを見て、興味のある様々なテーマについて調べてみましょう。
+たとえば、Kubernetes の初心者は、次のチャンネルに参加するのもよいでしょう。
+[`#kubernetes-novice`](https://kubernetes.slack.com/messages/kubernetes-novice)
```
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/troubleshooting.md:57]:
```suggestion
たとえば、Kubernetesの初心者は、[`#kubernetes-novice`](https://kubernetes.slack.com/messages/kubernetes-novice)に参加してみるのもよいでしょう。
```

File: content/ja/docs/tasks/debug-application-cluster/troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+別の例として、開発者は[`#kubernetes-dev`](https://kubernetes.slack.com/messages/kubernetes-dev)チャンネルに参加するとよいでしょう。
+
+また、多くの国別／現地語チャンネルがあります。これらのチャンネルに参加すれば、地域特有のサポートや情報を得ることができます。
```
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/troubleshooting.md:60]:
```suggestion
また、多くの国別/言語別チャンネルがあります。これらのチャンネルに参加すれば、地域特有のサポートや情報を得ることができます。
```

「国別」に対比するかたちの方が読みやすそうです

File: content/ja/docs/tasks/debug-application-cluster/troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+{{< /table >}}
+
+### フォーラム
+
+Kubernetesの公式フォーラムへの参加は大歓迎です[discuss.kubernetes.io](https://discuss.kubernetes.io)。
+
+### バグと機能の要望
+
+バグらしきものを発見した場合、または機能要望を出したい場合。
+GitHub課題追跡システム](https://github.com/kubernetes/kubernetes/issues)をご利用ください。
```
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/troubleshooting.md:89]:
```suggestion
バグらしきものを発見した場合、または機能要望を出したい場合、[GitHub課題追跡システム](https://github.com/kubernetes/kubernetes/issues)をご利用ください。
```

**Review by nasa9084 — CHANGES_REQUESTED** [review]

**Comment by ptux** [comment]
@nasa9084

Cloud you take a look at this PR?

**Comment by nasa9084** [comment]
see 2 comments I left before

**Comment by ptux** [comment]
@nasa9084 

fixed.

**Review by nasa9084 — COMMENTED** [review]
/lgtm

**Comment by ptux** [comment]
@inductor 

Cloud you take a look at this pr?

**Pull request merged**